dispatcher/http.go

```diff
@@ -0,0 +1,75 @@
+package dispatcher
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/whosonfirst/go-webhookd/v3"
+)
+
+func init() {
+
+	ctx := context.Background()
+	err := RegisterDispatcher(ctx, "http", NewHTTPDispatcher)
+
+	if err != nil {
+		panic(err)
+	}
+}
+
+// HTTPDispatcher implements the `webhookd.WebhookDispatcher` interface for dispatching messages to a `log.Logger` instance.
+type HTTPDispatcher struct {
+	webhookd.WebhookDispatcher
+	// logger is the `log.Logger` instance associated with the dispatcher.
+	logger *log.Logger
+	// url to send the message to
+	url string
+	// method to use when sending the message
+	method string
+}
+
+// NewHTTPDispatcher returns a new `HTTPDispatcher` instance configured by 'uri' in the form of:
+//
+//	http://
+//
+// Messasges are dispatched to the default `HTTP.Default()` instance.
+func NewHTTPDispatcher(ctx context.Context, uri string) (webhookd.WebhookDispatcher, error) {
+	logger := log.Default()
+	ctx = context.WithValue(ctx, ctxMethod{}, "POST")
+	return NewHTTPDispatcherWithLogger(ctx, logger)
+}
+
+// NewHTTPDispatcher returns a new `HTTPDispatcher` instance that dispatches messages to 'HTTPger'.
+func NewHTTPDispatcherWithLogger(ctx context.Context, logger *log.Logger) (webhookd.WebhookDispatcher, error) {
+
+	d := HTTPDispatcher{
+		logger: logger,
+		url:    ctx.Value(ctxUrl{}).(string),
```

Comment:
This should be configured using query parameters in `uri` or `uri` itself.

Reply:
Done